docs/checks.md

```diff
@@ -304,80 +304,92 @@ class HaltOnFailed(str, enum.Enum):
 Selecting the appropriate `halt_on` value lets you decide how much feedback you
 want to gather in a given check run.
 
-### CLI upgrades
+### Configuration
 
 All of the above changes are pretty hard to utilize without an interface. As
-such, the servo CLI has been upgraded with some new tricks:
-
-```console
-$ servo check --help
-Usage: servo check [OPTIONS] [CONNECTORS]...
-
-  Check that the servo is ready to run
-
-Options:
-  [CONNECTORS]...                 Connectors to check
-  -n, --name TEXT                 Filter by name  [default: False]
-  -i, --id TEXT                   Filter by ID  [default: False]
-  -t, --tag TEXT                  Filter by tag  [default: False]
-  -h, --halt-on-failed [requirement|check|never]
-                                  Halt running checks on a failure condition
-                                  [default: requirement]
+such, configuration for checks can be done in the checks section of the `servo.yaml`,
+as defined by the [ChecksConfiguration](../servo/configuration.py#L468) class.
+The checks configuration is not required explicitly, and if not specified will
+run with default options. Below is an example with all configurable options
+specified explicitly
+
+```servo.yaml
+    opsani_dev:
+      namespace: bank-of-anthos-opsani
+      deployment: frontend
+      container: frontend
+      service: frontend
+
+      envoy_sidecar_image: opsani/envoy-proxy:v1.20.1post1
+      cpu:
+        min: 250m
+        max: '3.0'
+      memory:
+        min: 128.0MiB
+        max: 3.0GiB
+    checks:
+      connectors: ['opsani-dev']
+      name: ['Connectivity to Kubernetes']
+      id: ['check_kubernetes_connectivity']
+      quiet: False
+      verbose: False
+      progressive: False
+      wait: 30m
+      delay: 10s
+      halt_on: critical
+      remedy: True
+      check_halting: False
+```
 
-  -v, --verbose                   Display verbose output  [default: False]
-  -q, --quiet                     Do not echo generated output to stdout
-                                  [default: False]
+By default, checks and any associated remedies run asynchronously, but remedies
+can be applied sequentially upon check failure by setting `check_halting` to True
 
-  --help                          Show this message and exit.
+```console
+    checks:
+      check_halting: True
 ```
 
-Results get aggregated and summarized by connector:
+Results from checks can be outputted into a table
+```console
+    checks:
+      progressive: True
```

Comment:
```suggestion
      progressive: False
```

This seems to be a misnomer that I wish I'd caught sooner but its too late to fix it now. I'll make a mental note to ensure its not carried over in the refactor 🙃 (also it has to be false to get the table)